src/modules/events/subscriber_handler.h

```diff
@@ -1,57 +0,0 @@
-#pragma once
-
-#include <uORB/uORB.h>
-#include <uORB/topics/battery_status.h>
-#include <uORB/topics/cpuload.h>
-#include <uORB/topics/vehicle_command.h>
-#include <uORB/topics/vehicle_status.h>
-#include <uORB/topics/vehicle_status_flags.h>
-#include <uORB/topics/vehicle_attitude.h>
-
-namespace events
-{
-
-/**
- * @class SubscriberHandler
- * Contains a list of uORB subscriptions and maintains their update state.
- */
-class SubscriberHandler
```

Comment:
I think the original idea of the `SubscriberHandler` was to save memory by not having all event tasks subscribe to the same uorb topics. But I don't know what the quantitative difference for this approach is.